ROADMAP.md

```diff
@@ -23,10 +23,9 @@ language. Running *multiple instances* of the daemon is also possible, and
 specially useful for testing purposes.
 
 When establishing connections, the daemon handles transport selection, security
-negotiation, and protocol and stream multiplexing. Streams are mapped 1:1 to
-local endpoints (e.g. unix sockets, shared-memory). Writes and reads to/from
-those endpoints are converted to writes and reads to/from the stream, allowing
-any application to interact with a libp2p network through simple, local IO.
+negotiation, and protocol and stream multiplexing. Streams are mapped 1:1 to multi-addresses. 
```

Comment:
Streams are mapped 1:1 to connections over streaming transports supported by go-multiaddr-net.

Reply:
Done